app/controllers/measurements_controller.rb

```diff
@@ -100,11 +114,11 @@ def new_notification
   end
 
   private
-    def authenticate
+  def authenticate
       if request.content_type.to_s.downcase != ENV['http_key'].downcase
         authenticate_or_request_with_http_basic('Administration') do |username, password|
-          username == ENV['http_basic_user'] && password == ENV['http_basic_password']
-        end
+        username == ENV['http_basic_user'] && password == ENV['http_basic_password']
```

Comment:
Did this work locally? We were having some trouble with it on production, therefore we removed it. If it's working keep it, if not leave it as it was.

Reply:
I didn't change anything there. When I try to resolved the authentication error, I erased de "!" next to downcase but I didn't save that. So, there was not change

Reply:
Ok, I see, I guess it's a matter of indentation (tabs and spaces added unintentionally to the code). Please undo these changes.